src/Badges.t.sol

```diff
@@ -1,63 +1,63 @@
 // SPDX-License-Identifier: CC0-1.0
-pragma solidity ^0.8.15;
+// pragma solidity ^0.8.15;
```

Comment:
Either let's update these tests or delete this file. My recommendation would be to unit test this in forge and integration test it in hardhat.

Reply:
Agreed. I'll take it out for now and make a ticket in linear to write proper unit tests